### DISCUSSION TITLE
Product Roadmap: Composer Support?

### DISCUSSION THREAD

**Discussion by LukeLeber** [community/community#20390] [Actions]
Hey there!

Is there any interest in adding support for PHP packages?  There is some precedence at https://packagist.com/docs/self-hosted/github-integration-setup, but it'd be _really nice_ if this could just work for everyone out of the box.

**Comment by Thijs-van-Drongelen** [comment]
We really want this feature. It's making a migration from gitlab difficult!

**Comment by michaelhue** [comment]
This would be much appreciated. Currently debating a switch to GitLab for their Composer support. I'd love to stay with GitHub though.
